modules/tensor_mechanics/src/scalarkernels/GeneralizedPlaneStrain.C

```diff
@@ -40,53 +40,26 @@ GeneralizedPlaneStrain::GeneralizedPlaneStrain(const InputParameters & parameter
     _gps(getUserObject<GeneralizedPlaneStrainUOInterface>("generalized_plane_strain")),
     _scalar_var_id(isParamValid("scalar_out_of_plane_strain_index")
                        ? getParam<unsigned int>("scalar_out_of_plane_strain_index")
-                       : 0)
+                       : 0),
+    _reference_residual_problem(dynamic_cast<ReferenceResidualProblem *>(
```

Comment:
Have you had a chance to talk to @bwspenc about this?

Reply:
Disregard, see https://github.com/idaholab/moose/pull/24249#issuecomment-1550511687